packages/serum/src/markets.json

```diff
@@ -1,1658 +1,68 @@
 [
   {
-    "address": "B37pZmwrwXHjpgvd9hHDAx1yeDsNevTnbbrN9W12BoGK",
```

Comment:
Nice, we should maybe use this as the canonical source for markets rather than the file in the resources repo.